lua/entities/terminator_nextbot_zambieglass/shared.lua

```diff
@@ -98,25 +115,78 @@ function ENT:Think()
 
 end
 
+// Default shard count
+ENT.iShards = 10
+
+local sound_Add = sound.Add
+sound_Add {
+	name = "nextbotZambies_GlassBreakA",
+	level = 85,
+	sound = {
+		"physics/glass/glass_largesheet_break1.wav",
+		"physics/glass/glass_largesheet_break2.wav",
+		"physics/glass/glass_largesheet_break3.wav"
+	}
+}
+sound_Add {
+	name = "nextbotZambies_GlassBreakB",
+	level = 80,
+	pitch = 110,
+	sound = {
+		"physics/glass/glass_sheet_break1.wav",
+		"physics/glass/glass_sheet_break2.wav",
+		"physics/glass/glass_sheet_break3.wav"
+	}
+}
+
+local SHARD_MODELS = {
+	"models/gibs/glass_shard01.mdl",
+	"models/gibs/glass_shard02.mdl",
+	"models/gibs/glass_shard03.mdl",
+	"models/gibs/glass_shard04.mdl",
+	"models/gibs/glass_shard05.mdl",
+	"models/gibs/glass_shard06.mdl",
+}
+local SHARD_MODELS_LENGTH = 6

```

Comment:
Just count the table with #table when we're picking the sounds, it only plays when the bots die, doesn't need to be optimized